test/image/mocks/ultra_zoom.json

```diff
@@ -0,0 +1,45 @@
+{
+    "data": [
```

Comment:
this mock looks like: 

![image](https://cloud.githubusercontent.com/assets/6675409/14002797/93113c0e-f124-11e5-8b71-7f2de4cfca64.png)

on `master`.